diesel/src/pg/types/network_address.rs

```diff
@@ -0,0 +1,194 @@
+extern crate ipnetwork;
+
+use std::io::prelude::*;
+use std::error::Error;
+use std::net::{Ipv4Addr, Ipv6Addr};
+
+use pg::Pg;
+use types::{self, ToSql, IsNull, FromSql};
+use self::ipnetwork::{IpNetwork,Ipv4Network,Ipv6Network};
+
+// const PGSQL_AF_INET: u8 = libc::AF_INET + 0;
+// const PGSQL_AF_INET6: u8 = libc::AF_INET + 1;
+const PGSQL_AF_INET: u8 = 2 + 0;
+const PGSQL_AF_INET6: u8 = 2 + 1;
```

Comment:
So I went and looked up **libc**'s implementation. https://doc.rust-lang.org/1.7.0/libc/constant.AF_INET.html. It is always 2. Now if you can point me to a list of all the architectures diesel supports I will happily go and verify the values of AF_INET for all of the architectures